src/models/global-leaderboard.js

```diff
@@ -0,0 +1,24 @@
+const { isString } = require('lodash');
+const BigNumber = require('bignumber.js');
+const { toLowerCase } = require('../utils');
+
+module.exports = class GlobalLeaderboard {
+  constructor(params) {
+    this.validate(params);
+    this.format(params);
+  }
+
+  validate(params) {
+    if (!isString(params.userAddress)) throw Error('userAddress must be a String');
+    if (!isString(params.investments)) throw Error('investments must be a String');
+    if (!isString(params.winnings)) throw Error('winnings must be a String');
+  }
+
+  format(params) {
+    this.userAddress = toLowerCase(params.userAddress);
+    this.investments = params.investments;
+    this.winnings = params.winnings;
+    const ratio = new BigNumber(this.winnings).dividedBy(new BigNumber(this.investments));
+    this.returnRatio = ratio.toString();
```

Comment:
do we need to convert to a string? wouldnt a number suffice here since were 0-100?

Reply:
The result will be a double type, think it better be string?

Reply:
We should keep the types as native as possible. Percentages are number types. The only reason we use Strings for other places is because it would overflow due to being too large.